inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java

```diff
@@ -440,16 +447,17 @@ public Boolean update(SinkRequest request, UserInfo opInfo) {
         }
         // update record
         SinkStatus nextStatus = null;
-        boolean streamSuccess = StreamStatus.CONFIG_SUCCESSFUL.getCode().equals(streamEntity.getStatus());
-        if (streamSuccess || StreamStatus.CONFIG_FAILED.getCode().equals(streamEntity.getStatus())) {
+        boolean enableConfig = StreamStatus.CONFIG_SUCCESSFUL.getCode().equals(streamEntity.getStatus())
+                || StreamStatus.CONFIG_FAILED.getCode().equals(streamEntity.getStatus());
+        if (enableConfig || StreamStatus.CONFIG_FAILED.getCode().equals(streamEntity.getStatus())) {
```

Comment:
Maybe `enableConfig || StreamStatus.CONFIG_FAILED.getCode().equals(streamEntity.getStatus())` change to 
`enableConfig`

Reply:
done